README.md

```diff
@@ -214,7 +214,7 @@ iTerm2 on a Mac -
 
 ![screenshot](assets/screenshot-mac.png)
 
-By default, downloads are saved under `videos/` unless specified by `-o` (output directory).
+By default, downloads are saved under `videos/` ( Not a system media Videos folder ), unless specified by `-o` (output directory).
```

Comment:
I'd prefer if you changed from videos to Destreamer/videos because that comment may leave some users confused on were it's the folder.
And maybe rephrase from 'not a system...' to 'not the system...'